index.html

```diff
@@ -1,13 +1,134 @@
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </div>
+    </nav>
+    <section id="header">
+      <div>
+        <blockquote>"Designing the future"</blockquote>
+        <h1 class="header">International <span>Technologies</span> Conference</h1>
+      </div>
+      <div class="introduction">
+        <p><span class="quote">"The most important technologies conference in the world"</span> <span class="author">Technologies Magazine</span></p>
+        <p>The IT Conference will be held in Paris. The most successful technology leaders will present on the future of technology.</p>
+      </div>
+      <div class="place">
+        <p class="date">2022.10.12 ~ 17 (saturday)</p>
+        <p class="salon">@ Le salon de paris, Paris, France</p>
+      </div>
+    </section>
+
+    <main>
+      <!-- Program Section -->
+      <section id="program">
+        <!-- cross bg -->
+        <div class="cross-bg">
+          <h2 class="program-heading">Main Program</h2>
+          <div class="line"></div>
+
+          <!-- programs-container -->
+          <div class="programs-container">
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_01.png" alt="lecture icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Listen to speakers from various tech companies about the messages of tech future.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_02.png" alt="exhibition icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Let's get to know the creations of artists from various fields who share the open spirit of ITC.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_03.png" alt="forum icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">We have time to share our thoughts and opinions with experts by topic.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_04.png" alt="workshop icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">You can create your own creations using open source code instead of just looking at them.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Create free networking opportunities with ITC staff from around the world.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+
+        </div>
+        <!-- cross bg -->
+      </section>
```

Comment:
- [ ] Great job here 💯but you I think you forgot to add the button, kindly add it to look similar to the original desig👌

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/190325486-01ff913c-2b7f-4ada-ae04-7356cb893778.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/190325994-2185668c-dbb8-4d39-834f-e69c7c251f94.png" width="500" /> |

- NOTE: this button on the mobile version/size should look like this ⬇

![image](https://user-images.githubusercontent.com/45231276/190326179-3ac867ac-e4fd-43c8-8973-9d348d21e848.png)